package.json

```diff
@@ -78,7 +78,7 @@
     "rollup-plugin-dts": "^6.1.0",
     "rollup-plugin-esbuild": "^6.1.1",
     "ts-jest": "^29.2.2",
-    "tslint": "^6.1.3",
+    "tslib": "^2.8.1",
```

Comment:
The build was indirectly depending on tslib, and removing tslint make it so it could not build.